components/org.wso2.carbon.business.rules.templates.editor.core/package-lock.json

```diff
@@ -4550,791 +4549,6 @@
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz",
       "integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8="
     },
-    "fsevents": {
```

Comment:
Let's not commit this file since you have not done any changes to package.json of components/org.wso2.carbon.business.rules.templates.editor.core